app/javascript/components/dropdowns/Dropdown.tsx

```diff
@@ -1,9 +1,9 @@
-import React, { useState, useEffect, useRef } from 'react'
+import React, { useState, useEffect, useRef, useMemo } from 'react'
 import { usePanel } from '../../hooks/use-panel'
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
Good idea to rename!

Reply:
![image](https://user-images.githubusercontent.com/1964376/106750094-ac92ed80-6627-11eb-9839-eb51c9d70d46.png)

From the docs 🖌️